### ISSUE TITLE
Image Cards with Hover Effect

### ISSUE THREAD

**Issue by AnanyaM007** [Web3-Bharat-ITER/UI-Component-Library#28]
Hey @shashank-iter !
I would like to add a new UI component of image cards with hover effects. Can you assign it to me?

**Comment by hafidzzakky** [comment]
Hey I am interested in making a navbar .
Can you assign me ?

**Comment by shashank-iter** [comment]
@AnanyaM007 You can definitely work on that. Just add a screenshot of the work while making the PR. Happy Hacktober Month 😀.

**Comment by shashank-iter** [comment]
@hafidzzakky yes you can work on that, kindly read the contribution guidelines and add a screenshot of your work while making a PR.

**Comment by shashank-iter** [comment]
@AnanyaM007 You previously opened an issue for making an image card. You can work on that too and make a separate PR for that. Add a screenshot of your work too.

**Comment by AnanyaM007** [comment]
Alright thanks! I'll add both of them.

**Comment by AnanyaM007** [comment]
hey! I've added the image card. can you verify that..

**Comment by kasib12** [comment]
Hey @shashank-iter @Jyo561Python can you please assign this issue to me? I think I can work on the problem and provide a much better solution for it.

**Comment by shashank-iter** [comment]
@kasib12 assigned it to you. You can start working.
